dynawo/3rdParty/suitesparse/patch/MSVC/suitesparse.patch

```diff
@@ -1,7 +1,8 @@
-diff -urpN SuiteSparse/CMakeLists.txt SuiteSparse.patch/CMakeLists.txt
---- SuiteSparse/CMakeLists.txt	2019-10-08 13:37:10.745154612 +0200
-+++ SuiteSparse.patch/CMakeLists.txt	2019-10-09 10:37:06.523791858 +0200
-@@ -1,8 +1,22 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8067f04..a0d90ba 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,5 +1,11 @@
  cmake_minimum_required(VERSION 3.4.3 FATAL_ERROR)
  
 +if(${CMAKE_VERSION} VERSION_GREATER "3.15.0")
```

Comment:
this policy is only set for msvc static runtime so it is useless and then I think this whole patch becomes useless

Reply:
OK